examples/README.md

```diff
@@ -0,0 +1,7 @@
+# Example Connectors
```

Comment:
# Example Connectors

The examples in this directory show various ways to get started with building Nakji connectors.

- The Ethereum example uses the base Nakji connector library to send blocks and transactions to Nakji.
- The Compound example uses the [ethereum chain-specific connector library](../chain) to send ethereum events to Nakji.

These connector examples in this directory are simplified and commented, and may not handle everything the production connector does. The goal is just to demonstrate the concept of a connector for those who like to read code over reading documentation.

For those who want to see full connectors that are actually running in production as additional examples, please refer to github.com/nakji-network/connectors.